### ISSUE TITLE
Request url broken with @ActivationRequestParameters

### ISSUE THREAD

**Issue by ccburnley** [exanpe/exanpe-t5-lib#18]
The urls generated by the javascript don't account for their being parameters already part of the URL. 

For example: 

exanpe-t5-lib.js : 2430

```
        this.url + "?" + Exanpe.PasswordStrengthChecker.PARAM_NAME + "=" + password,
```

the URL if the page has activation request parameters already has a "?" in it and would in this case need to be a "&".

**Comment by lguerin** [comment]
I have pushed a fix. Thank for the issue report.

**Comment by ccburnley** [comment]
Works great for me, thanks. 

BTW, where can I get snapshots of 1.3.0 ?

**Comment by lguerin** [comment]
I have just created a bug fix release : exanpe-t5-lib-1.2.1.
This version should be available into Maven central in few hours.
